### ISSUE TITLE
Footer center does not show when using header-html

### ISSUE THREAD

**Issue by jbrumwell** [wkhtmltopdf/wkhtmltopdf#3064]
If I include the `header-html` option and `footer-center` the footer does not show. If I omit the `header-html` option the footer shows up.

**Comment by jbrumwell** [comment]
Looks to be any footer commands;

`wkhtmltopdf --quiet --print-media-type --footer-center "[page]" --header-spacing 5 --footer-spacing 5 --page-size "letter" --orientation "Portrait" --viewport-size "1280x1024" --header-html "/tmp/tmp-9543Jiil4ZT2DGZI.html"`

Results in no footer, if you remove the header-html option the footer generates as expected

**Comment by LuizFeReis** [comment]
Same problem here, can't use Header and Footer together, only the header shows up.

Windows 7 64 (wkhtmltopdf 0.12.2.2 (with patched qt))

**Comment by Schtorsk** [comment]
I thought I had the same problem too. But I really needed to combine` header-html` with `footer-center` so I persevered. Turns out it was just a margin problem.

This does not work:
`--footer-center "[page]/[toPage]" --header-html "someUrlToHtmlFile"`

This does:
`--footer-center "[page]/[toPage]" --header-html "someUrlToHtmlFile" --margin-top 35mm --margin-bottom 15mm"`

Maybe you're facing the same problem?
